jetty-http/src/test/java/org/eclipse/jetty/http/pathmap/PathMappingsTest.java

```diff
@@ -461,4 +462,18 @@ public void testRemoveServletPathSpec()
         assertThat(p.remove(new ServletPathSpec("/a/b/c")), is(true));
         assertThat(p.remove(new ServletPathSpec("/a/b/c")), is(false));
     }
+
+    @Test
+    public void testAsPathSpec()
+    {
+        assertThat(PathMappings.asPathSpec(""), instanceOf(ServletPathSpec.class));
```

Comment:
What about `asPathSpec(null)`, should that be interpreted as ROOT too?

Reply:
No,  it is very precisely the empty string that matches "/" (and "/" matches everything - go figure!)